analytics/google/experience-accelerator/src/index.ts

```diff
@@ -1,121 +1,4 @@
-export class GAClient {
-    queue: any[] = [];
-    interval: number = 50;
+import { GAClient } from "./GAClient";
+import { GtagClient } from "./GtagClient";
```

Comment:
Not necessary to change:
```suggestion
export { GAClient } from "./GAClient";
export { GtagClient } from "./GtagClient";
```